invenio_communities/assets/semantic-ui/less/invenio_communities/theme.less

```diff
@@ -7,9 +7,8 @@
  */
 
 .community-header {
-  background: #F8F8F9;
+  background: #f8f8f9;
```

Comment:
Why aren't we defining here at least the colors as a variable? It would make it easier to maintain, but maybe there is a reason for not doing it.

Reply:
Having a variable is necessary I agree :)

Reply:
Agreed with @kpsherva to fix this in next issues (https://github.com/inveniosoftware/invenio-theme/issues/265)

Reply:
Ok, apart from that LGTM!